lib/utils/task-option-validator.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+var di = require('di');
+var path = require('path');
+
+module.exports = taskOptionValidatorFactory;
+
+di.annotate(taskOptionValidatorFactory, new di.Provide('TaskOption.Validator'));
+di.annotate(taskOptionValidatorFactory, new di.Inject(
+    'JsonSchemaValidator',
+    'FileLoader',
+    '_',
+    'Util'
+));
+
+function taskOptionValidatorFactory(
+    JsonSchemaValidator,
+    FileLoader,
+    _,
+    util
+) {
+    var TaskSchemaFolder = path.resolve(__dirname, '../../lib/task-data/schemas');
+    var MetaSchemaFileName = 'rackhd-task-schema.json';
+    var ContextPattern = /\{\{[\s\S]*context\.[\s\S]*\}\}/;
```

Comment:
Thanks @benbp . 
Looks like there is no necessary to expose these var outside. So I would update according to @yyscamper  's suggestion.